### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
# Bookstore: initialize the project with components 🎉 ✔️ 🎉 ✔️ 🎉 
- Use React Router.
- Use React components.

### CODE REVIEW

File: src/components/BookList.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import Header from './Header';
+
+function BookList() {
+  return (
+    <>
+      <Header />
+      <div>Book and Auther Name</div>
+      <div>Book and Auther Name</div>
+      <div>Book and Auther Name</div>
+      <div>Book and Auther Name</div>
```
Comment (codepantha) [inline src/components/BookList.jsx]: - [ ] You have a `BookList` component which is good, but you also need a `<Book />` component which is going to take the book data as props and render the author and the title.

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <MainDiv />
+    </>
```
Comment (codepantha) [inline src/App.js:7]: - [ ] [OPTIONAL] In my opinion `<MainDiv />` is not a good name for a component. It is vague and doesn't indicate what the component might be doing. I have also looked at the <MainDiv /> component and I'm of the opinion that everything going on in that component should go in the <App /> component instead. In other words, the <MainDiv /> component is redundant. Please take this as an optional suggestion 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @Nizamuddin4493 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights ⭐ 
✔️ Descriptive PR
✔️ Meaningful commit messages
✔️ Linter checks

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @Nizamuddin4493, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**